deisctl/units/deis-mesos-master.service

```diff
@@ -13,7 +13,7 @@ ExecStartPre=-/usr/bin/docker rm deis-mesos-master
 ExecStartPre=/bin/sh -c "docker inspect deis-mesos-master-data >/dev/null 2>&1 || docker run --name deis-mesos-master-data -v /tmp/mesos-master alpine:3.1 /bin/true"
 ExecStartPre=/bin/sh -c "IMAGE=`/run/deis/bin/get_image /deis/mesos-master` && docker history $IMAGE >/dev/null 2>&1 || docker pull $IMAGE"
 ExecStart=/usr/bin/sh -c "IMAGE=`/run/deis/bin/get_image /deis/mesos-master` && docker run --volumes-from=deis-mesos-master-data --name=deis-mesos-master --privileged --net=host -e HOST=$COREOS_PRIVATE_IPV4 $IMAGE"
-ExecStop=-/usr/bin/docker stop deis-mesos-master
+ExecStop=/usr/bin/docker stop deis-mesos-master
```

Comment:
Ok.  Good call.  I was trying to follow the example in the Fleet documentation.  But I agree with what you're saying.  I'll fix that.